slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/ActionTest.scala

```diff
@@ -358,6 +359,35 @@ class ActionTest extends AsyncTest[RelationalTestDB] {
     for {
       _ <- s1s.schema.create
       _ <- s2s.schema.dropIfExists
+      _ <- s2s.schema.dropIfExists
+    } yield ()
+  }
+
+  def testDropIfExistsWithPK = {
```

Comment:
Why does having a PK require its own test? And how does that relate to the topic of this PR?

Why not have a simple test that runs dropIfExists on a simple table that hasn't been created?

Reply:
> Why does having a PK require its own test? And how does that relate to the topic of this PR?

The original problem was that `dropTableIfExists` would fail when there is not a table with PK. This test is to make sure the problem is resolved.


> Why not have a simple test that runs dropIfExists on a simple table that hasn't been created?

dcf8ac4